.gitignore

```diff
@@ -0,0 +1,4 @@
+
+githubToken.js
+
+node_modules/
```

Comment:
Great adding .gitignore! Now, you just need to delete the node_modules you pushed up earlier....